web-dev/src/services/users-service.js

```diff
@@ -4,23 +4,28 @@ const BASE_URL = "http://localhost:4000/api"
 const LOGIN_API = `${BASE_URL}/login`;
 const USERS_API = `${BASE_URL}/users`;
 
+const api = axios.create({
+  withCredentials: true
+});
+
+
 export const createUser = (user) =>
-  axios.post(`${USERS_API}`, user).then(response => response.data);
+  api.post(`${USERS_API}`, user).then(response => response.data);
 
 export const findAllUsers = () =>
-  axios.get(USERS_API).then(response => response.data);
+  api.get(USERS_API).then(response => response.data);
 
 export const findUserById = (uid) =>
-  axios.get(`${USERS_API}/${uid}`).then(response => response.data);
+  api.get(`${USERS_API}/${uid}`).then(response => response.data);
 
 export const deleteUser = (uid) =>
-  axios.delete(`${USERS_API}/${uid}`).then(response => response.data);
+  api.delete(`${USERS_API}/${uid}`).then(response => response.data);
 
 export const deleteUsersByUsername = (username) =>
-  axios.get(`${USERS_API}/username/${username}/delete`).then(response => response.data);
+  api.get(`${USERS_API}/username/${username}/delete`).then(response => response.data);
 
 export const findUserByCredentials = (credentials) =>
-  axios.post(`${LOGIN_API}`, credentials).then(response => response.data);
+  api.post(`${LOGIN_API}`, credentials).then(response => response.data);
 
 const service = {
```

Comment:
I have NO idea what this is or why it's here. I will see if I can remove this later.